langchain/src/llms/tests/openai.int.test.ts

```diff
@@ -180,3 +180,27 @@ test("Test OpenAI prompt value", async () => {
   }
   console.log({ res });
 });
+
+test("Test PromptLayerOpenAI returns promptLayerID iff returnPromptLayerID=true", async () => {
```

Comment:
Fix typo. Also this should be in its own integration test, not part of the existing one.